lib/UnoCore/Targets/Android/app/build.gradle

```diff
@@ -133,8 +147,32 @@ android {
 #elif @(Project.Android.Key.Store:IsSet)
             signingConfig = signingConfigs.release
 #endif
-            minifyEnabled = false
-            proguardFiles 'proguard-rules.pro'
+
+#if @(Project.Android.Optimize.MinifyEnabled:IsSet)
+            // Enables code shrinking, obfuscation, and optimization for only
+            // your project's release build type.
+            minifyEnabled @(Project.Android.Optimize.MinifyEnabled:ToLower)
+#else
+            minifyEnabled true
+#endif
+
+#if @(Project.Android.Optimize.ShrinkResources:IsSet)
+            // Enables resource shrinking, which is performed by the
+            // Android Gradle plugin.
+            shrinkResources @(Project.Android.Optimize.ShrinkResources:ToLower)
+#else
+            shrinkResources true
+#endif
+            // Includes the default ProGuard rules files that are packaged with
+            // the Android Gradle plugin. To learn more, go to the section about
+            // R8 configuration files.
+            proguardFiles getDefaultProguardFile(
+#if @(Project.Android.Optimize.DefaultAndroidProGuard:IsSet)
```

Comment:
Naming: I don't think we should repeat `Android` in this property name. I suggest `Project.Android.Optimize.DefaultProguard:IsSet`.